package.json

```diff
@@ -37,7 +37,7 @@
     "react-hook-form": "^7.50.0",
     "react-markdown": "^9.0.1",
     "react-nocomment-fork": "^0.5.0",
-    "react-nostr": "^0.0.6",
+    "react-nostr": "link:react-nostr",
```

Comment:
Can we publish a proper release of `react-nostr` and just bump the version here?